.travis.yml

```diff
@@ -21,6 +21,8 @@ matrix:
         - TESTING_ON_TRAVIS=1
 install:
   - pip3 install -r requirements-testing.txt
-script: python3 -m coverage run tests.py || python -m coverage run tests.py
+script: 
+  - python3 -m coverage run -m unittest -v tests.py || python -m coverage run -m unittest -v tests.py
+  - python3 -m coverage report -m || python -m coverage report -m
```

Comment:
Unsure whats the intention here, we already generate coverage data and `coveralls` is consuming that `after_success`.  See https://coveralls.io/github/abhinavsingh/proxy.py for existing reports.

Reply:
I want to see the coverage details, for example which lines is not covered exactly, but i have not found the information in coveralls.io. So I add the command temporarily.

Reply:
We can run `make coverage` locally to get line-by-line coverage.

Also, even if we generate a coverage on travis, does coveralls.io shows that to you?

Reply:
Also see https://coveralls.io/builds/25936980/source?filename=proxy.py where you can get line-by-line coverage report.  These were generated with existing integration IIUC.

Reply:
Example https://coveralls.io/builds/25941085 this is the coverage report for the latest build of your PR.

Reply:
The coverage locally is 89%, so I want to figure out  the difference.

> does coveralls.io shows that to you?

Yes, we can see the output in the log page.

Reply:
> Also see https://coveralls.io/builds/25936980/source?filename=proxy.py where you can get line-by-line coverage report. These were generated with existing integration IIUC.

Got it. I'll remove the this change.

Reply:
> Yes, we can see the output in the log page.

Great.  Here is a link to all historical builds https://coveralls.io/repos/272015/builds  As you can see, these line-by-line coverage reports are being generated since [day 1 of coveralls.io integration](https://coveralls.io/builds/25333644).

Hence, I don't see why we should add more commands to `.travis.yml` file.  I am unsure about what additional value does it provide that already doesn't exists :)  Wdyt?

Reply:
> > Also see https://coveralls.io/builds/25936980/source?filename=proxy.py where you can get line-by-line coverage report. These were generated with existing integration IIUC.
> 
> Got it. I'll remove the this change.

No worries.  Thanks for poking me, I just discovered these myself 👍

Reply:
> I don't see why we should add more commands to .travis.yml file

You're right, I did find the line-by-line report before.